### PR TITLE
Integrated poplar into <gaia-fast-list> (closes #39)

### LINKED ISSUE #39: Integrate Poplar into <gaia-fast-list>

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@etiennesegonzac r?

**Comment by etiennesegonzac** [comment]
Nit: might be a good time to add the "magical webcomponent" permission to the example's manifests.

File: fast-list.js [hunk 1]
```diff
@@ -24,25 +29,27 @@ exports.scheduler = schedule;
...
-  this.container.style.overflowY = 'scroll';
-
-  this.list = source.list;
-  if (!this.list) {
-    this.list = document.createElement('ul');
-    this.container.appendChild(this.list);
+  // List elements aren't required
+  if (!source.list) {
+    source.list = document.createElement('ul');
+    source.container.appendChild(source.list);
```
Comment (etiennesegonzac) [inline fast-list.js:37]: can be a follow up since it's not introduced in the PR, but we should maybe cover some of the "source fallbacks" with unit tests.

Comment (wilsonpage) [inline fast-list.js:37]: Can you give some more detail?

Comment (etiennesegonzac) [inline fast-list.js:37]: It's actually the opposite, we're testing the !source.list case, but I don't think we're testing using the provided list.

**Pull request merged**